templates/cookies/cookies_settings.twig

```diff
@@ -0,0 +1,66 @@
+{% set necessary_cookies_name = cookies.settings_copy.necessary_cookies_name %}
+{% set necessary_cookies_description = cookies.settings_copy.necessary_cookies_description %}
+{% set analytical_cookies_name = cookies.settings_copy.analytical_cookies_name %}
+{% set analytical_cookies_description = cookies.settings_copy.analytical_cookies_description %}
+{% set all_cookies_name = cookies.settings_copy.all_cookies_name %}
+{% set all_cookies_description = cookies.settings_copy.all_cookies_description %}
+
+<div class="d-none cookies-settings">
+	<div class="close-cookies-settings"></div>
+	<div class="cookies-settings-header">
+		<h4>{{__('Manage your cookies preferences', 'planet4-master-theme')}}</h4>
+		<p>{{__('Please select which cookies you are willing to store.', 'planet4-master-theme')}}</p>
+	</div>
+	<div class="cookies-settings-details">
+		{% if necessary_cookies_name is not empty %}
+			<div class="custom-control-description">{{ necessary_cookies_name }} <span class="always-enabled">{{__('Always enabled', 'planet4-master-theme')}}</span></div>
+			{% if necessary_cookies_description is not empty %}
+				<p>{{ necessary_cookies_description }}</p>
```

Comment:
The descriptions don't display the allowed html properly (I assume you need the same filter `|e('wp_kses_post')|raw` used for cookies.text)